scripts/templates.py

```diff
@@ -86,7 +87,11 @@ def read_yaml_files(yaml_dirs, name: str = "config.yaml"):
 
 
 def process_config_files(dirs, yaml_data, build_dir="./build"):
-    Path(build_dir).mkdir(exist_ok=True)
+    
+    path = Path(build_dir)
+    if path.exists():
+        shutil.rmtree(path)
```

Comment:
if someone misconfigures their build dir, you nuke it? :D

Reply:
The idea was to ensure deterministic behaviour. If we don't clean the directory, you may have unwanted leftover artifacts from previous build. I partly agree that a `--force` or `--clean` flag is slightly better.